qtranslate.php

```diff
@@ -54,7 +54,7 @@
 	ja by Brian Parker
 	nl by RobV
 	pt by netolazaro
-	pt-br by Pedro Mendonça
+	pt-pt by Pedro Mendonça
```

Comment:
Ok, let's look just the 'pt' and 'pt-br' codes.

pt (Portugal)
pt-br (Brazil)

'qtranslate.php' use to have only one of this: 'pt'
The 'pt' that existed had the brazilian flag and texts. I've just changed it to the correct 'pt-br' maintaining the current brazilian flag and texts.
Then I've added the 'pt' (from Portugal) now with the portuguese flag and texts.

Was this more clear now?